### ISSUE TITLE
[SUGGESTION] Crew member NPCs

### ISSUE THREAD

**Issue by OverSavior** [tsunamayo/Skywanderers#1461]
The recent kickstarter update where general and turret AI is mentioned made me think about one thing. A cool feature would be being able to hire crew on stations or anywhere else. Or, to make it more interesting, being able to find survivors in space that may then choose to join you. How I imagine it is that each crew member will have certain skills, basically things they're good at. It can be either turret/drone control, repairs, scanning, power managing or some other thing depending on mechanics that will be implemented in the future (coughcoughspaceinfantrycough). They may also gain experience and become better at doing those things or perhaps learn new skills. The more skilled the crew member is, the better he does its job and the more you'll have to pay him (unless you're lucky and they decided to travel with you for free or because you saved their lives). Also, if the crew member gets wounded they may die and you'll have to search for a new one. This could open for many possibilities. Players could more or less effectively control larger ships and in general this could probably save some time if they were to repair their ship. Even some lore could be involved here! Although, such feature would most likely be really hard to implement. Even the way NPCs would havigate the ship. But certainly it would be a lot of fun. Just imagine loosing one of your crew members on a large ship and then struggling getting it to the nearest station.

**Comment by Zeuscp1** [comment]
This. Would be a game changer feature. Being able to have a crew that helps you out with the ship performance, without being overpowered would be really fun. I don't know if it's okey to mention another game here, but look at Conan Exiles Thralls system, something like that but instead of being slaves, they would be hired, or recruited. Also there could be the option of having 3 types, slaves (poor performance and can be sold or try to escape if you don't keep them in check with a device or something.) Recruits (stay with you permanently as companions, and you shall only have X amount demending of another factor like clan size). And mercenaries like, that you pay them and they will work for you for X time.

**Comment by tsunamayo** [comment]
Yes thats exactly the way I want to go, not sure when I will have time to really tackle this, first I need to make big progress on the gameplay side!
